### PR TITLE
keep build dir

### PR DESCRIPTION
**Pull request by n0izn0iz**

### CODE REVIEW

**Comment by n0izn0iz** [comment]
Random CI fails.......

**Comment by n0izn0iz** [comment]
Des tests renvoyaient invalid opcode, j'ai relancé le build sans changer le code et ça passe........

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,5 @@
 node_modules
 npm-debug.log
-build
```
Comment (louptheron) [inline .gitignore:3]: why do we need the build folder ? We want to build .sol from the core no ?

**Review by louptheron — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (n0izn0iz) [reply .gitignore:3]: no, before publish in contracts actually I think
http://truffleframework.com/docs/getting_started/packages-npm

Comment (n0izn0iz) [reply .gitignore:3]: npm don't copy it if it is in .gitignore

Comment (louptheron) [reply .gitignore:3]: Right for the .gitignore...  So do you want to `truffle compile` before using `readContracts` from the core ? Seems weird

Comment (n0izn0iz) [reply .gitignore:3]: no, before npm publish or run it in local dir before npm install in core so contracts get copied with the built json then contracts index expose the .jsons

Comment (louptheron) [reply .gitignore:3]: Also not so clever but let's start with it

Comment (n0izn0iz) [reply .gitignore:3]: thats the ethereum packaging format.....

Comment (n0izn0iz) [reply .gitignore:3]: core depends on the built contracts not the source

Comment (n0izn0iz) [reply .gitignore:3]: we can't build the contracts in every client that needs them, it's slow

**Comment by n0izn0iz** [comment]
Par contre les build qui fail une fois sur deux c'est extremement relou :(

**Review by louptheron — APPROVED** [review]

**Pull request merged**